patient-search/ui/src/components/Table/Table.tsx

```diff
@@ -0,0 +1,91 @@
+import { Button, Icon, Table, Pagination } from '@trussworks/react-uswds';
+import React from 'react';
+
+type TableContentProps = {
+    tableHeader?: string;
+    tableSubHeader?: React.ReactNode | React.ReactNode[] | string;
+    tableHead: { name: string; sortable: boolean }[];
+    tableBody: any[];
```

Comment:
Can we have type for tableBody here. looks like one of the property is title

Reply:
Yes good idea